src/components/FilterBar/FilterDisplay/Inputs/SelectInput.react.js

```diff
@@ -1,8 +1,16 @@
 export class SelectInput extends React.Component {
-  constructor(props) {
-    super(props);
+  constructor(props, context) {
+    super(props, context);
 
-    this.state = {value: props.value};
+    const options = context.filterBarStore.getFilter(this.props.filterUid).options;
+    const passedValue = props.value;
+
+    if (passedValue === "" || passedValue === undefined) {
+      this.state = {value: options[0].value};
+      context.filterBarActor.updateFilter(props.filterUid, "value", options[0].value);
```

Comment:
I really hate it, very impure. I need to refactor the store, but that's a larger body of work than I felt I can get to in this ticket. If it blocks this PR, I'll start working on that now?